build_harmonizer.rs

```diff
@@ -25,21 +25,34 @@ fn main() {
     create_snapshot().expect("unable to create v8 snapshot: query_runtime.snap");
 }
 
-// runs `npm ci` && `npm run build` in the current `harmonizer-x` workspace crate
+// runs `npm install` && `npm run build` in the current `harmonizer-x` workspace crate
 fn bundle_for_deno() {
     let npm = which::which("npm").expect("You must have npm installed to build this crate.");
     let current_dir = std::env::current_dir().unwrap();
 
-    println!(
-        "cargo:warning=running `npm ci` in {}",
-        &current_dir.display()
-    );
-    assert!(Command::new(&npm)
-        .current_dir(&current_dir)
-        .args(&["ci"])
-        .status()
-        .expect("Could not get status of `npm ci`")
-        .success());
+    if cfg!(debug_assertions) {
+        println!(
+            "cargo:warning=running `npm install` in {}",
+            &current_dir.display()
+        );
+        assert!(Command::new(&npm)
+            .current_dir(&current_dir)
+            .args(&["install"])
+            .status()
+            .expect("Could not get status of `npm install`")
+            .success());
+    } else {
+        println!(
+            "cargo:warning=running `npm ci` in {}",
+            &current_dir.display()
+        );
+        assert!(Command::new(&npm)
+            .current_dir(&current_dir)
+            .args(&["ci"])
+            .status()
+            .expect("Could not get status of `npm install`")
```

Comment:
Not sure if this intentional or not.

```suggestion
            .expect("Could not get status of `npm ci`")
```

Reply:
good catch ty